### PR TITLE
Dev fe

### PR DESCRIPTION
**Pull request by josiahphua**

### CODE REVIEW

File: src/components/browse/Browse.jsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ import { InventoryContext } from '../../store/inventory/inventory';
 import {addToCart} from '../../store/actions';
 
 // DISPATCH addToCart function here
-//effy sucks
```
Comment (ZAF07) [inline src/components/browse/Browse.jsx:19]: Nice work jia.. Impressive code

**Review by ZAF07 — COMMENTED** [review]
Good job. This is really impressive!!

File: src/components/browse/Browse.jsx (hunk 1 above)
Comment (josiahphua) [reply src/components/browse/Browse.jsx:19]: Yes thank you sir.

**Review by ZAF07 — APPROVED** [review]
Nice go ahead and merge. road to startup founders

**Pull request merged**